src/hooks/wallets/useOnboard.ts

```diff
@@ -19,7 +19,7 @@ export type ConnectedWallet = {
   provider: EIP1193Provider
 }
 
-const lastWalletStorage = localItem<string>('lastWallet')
+export const lastWalletStorage = localItem<string>('lastWallet')
```

Comment:
Could you make a function `forgetLastWallet` and export that instead of exporting the storage?